index.html

```diff
@@ -5,9 +5,21 @@
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title id="title"></title>
    <link rel="stylesheet" href="styles.css" />
+   <title>To-do list</title>
   </head>
   <body>
-    
+      <div class="to-do-app">
+        <h2 class="blocks">To-do list</h2>
+        <div class="blocks" id="input-field">
```

Comment:
I see that you use same class for H2 and Div, I would suggest tot divide it or change class naming, as it is not convenient.